tools/case/architecture.plugin/.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>architecture.plugin</name>
+	<name>com.collins.fmw.cyres.architecture.plugin</name>
```

Comment:
Could we rename package prefix "cyres" to "staircase" and "case" directory to "staircase"?

Reply:
Let's leave this pull request as it is.  As this has to do with "branding" of our tools, please discuss with the project PI (Darren) as to how it should be done.